meetupdao/UI/web3/proposals.js

```diff
@@ -152,7 +152,7 @@ function getProposals(){
 		var a6 = '</span></p></font></div><BR>';
 
     if (result[8].s < 0) {
-        currentStatus = -currentStatus;
+        currentStatus = (-1 * currentStatus);

```

Comment:
This should be `NOT` of boolean. Multiplying by -1 won't work here

Reply:
nice catch